vite.config.ts

```diff
@@ -1,7 +1,15 @@
 import { defineConfig } from 'vite'
 import react from '@vitejs/plugin-react'
 
+const path = require('path');
```

Comment:
do not mix `import` and `require` in the same file. always use `import` in TS files.